.kitchen.vagrant.yml

```diff
@@ -23,6 +32,10 @@ platforms:
   driver_config:
     box: opscode-ubuntu-14.04
     box_url: https://opscode-vm-bento.s3.amazonaws.com/vagrant/virtualbox/opscode_ubuntu-14.04_chef-provisionerless.box
+- name: ubuntu-16.04
+  driver_config:
+    box: opscode-ubuntu-16.04
```

Comment:
I'd like to focus on the docker part since I simply copied the vagrant-configuration and did not change it. I'll change it in another PR.